examples/v2/common/python/container_vm.py

```diff
@@ -69,7 +69,11 @@ def GenerateConfig(context):
           'network': GlobalComputeUrl(context.env['project'],
                                       'networks',
                                       'default')
-          }]
+          }],
+        'serviceAccounts': [{
```

Comment:
why is this change required?

Reply:
the container are not starting without it. They produce an error like this:
`docker: Error response from daemon: Failed to initialize logging driver: unable to connect or authenticate with Google Cloud Logging: Post https://logging.googleapis.com/v1beta3/projects/qwiklabs-gcp-f29e3dcb8073a1d0/logs/gcplogs-docker-driver/entries:write?alt=json: metadata: GCE metadata "instance/service-accounts/default/token" not defined.`